recipes/lammps/a100/environments.yaml

```diff
@@ -0,0 +1,36 @@
+lammps:
+  compiler:
+      - toolchain: gcc
+        spec: gcc@11
+  mpi:
+      spec: cray-mpich
+      gpu: cuda
+  unify: when_possible
+  specs:
+  - cmake
+  - cuda@11.8
+  - hwloc
+  - python@3.10
+  - vtk
+  - kokkos
+  - blas
+  - lapack
+  - fftw-api
+  - lammps@develop +cuda
```

Comment:
Is there a specific version that we should target instead of `@develop`?

Reply:
I was getting build errors for the other specific versions. `@develop` didn't give any. Investigating further.